tutorials/introductory/sample_plots.py

```diff
@@ -36,12 +36,46 @@
 
    Subplot
 
+.. _screenshots_images_demo:
+
+Images
+======
+
+Matplotlib can display images (assuming equal-spaced, equal-aspect-ratio
+horizontal dimensions) using the :func:`~matplotlib.pyplot.imshow` function.
```

Comment:
1. you don't actually need equal aspect ratio
2. something seems to be missing before or after "horizontal dimensions"?